res/ps_border.fs.glsl

```diff
@@ -56,6 +56,59 @@ vec4 draw_dotted_edge() {
   return mix(white, circleColor, circleColor.a);
 }
 
+vec4 draw_double_edge(float pos, float len) {
+  // Devided border to 3 parts, draw color on first and third part,
+  // leave second part blank.
+  float one_third_len = len / 3.0;
+
+  // The original algorithm should be check if pos in the blank part.
+  // That is, checking (1/3 * len) < pos < (2/3 * len)
+  //
+  // The idea here is, offset pos by (1/3 * len) then mod it by len.
+  // And if this value is lesser than (2/3 * len) then we're in
+  // blank part. Thus, we can use only 1 check instead of 2.
+  float compared_pos = mod(pos + one_third_len, len);
```

Comment:
How does this work? Let's say I have a length of 100 and I want to see of pos 50 is in the blank part.

So 0- 33 - Filled
33 - 66 - Blank
66 - 100 filled

(1/3) * len = 33 here

So offset 50 would be:
50 + 33 = 83
83 % 100 = 83

This doesn't match the comment, but it does match the code. In the code you're checking that compared_pos is greater than 2/3, which if true, means its in the blank part. Please fix the comment. I think should be "greater than (2/3 * len).

nit: I'd rather do the most obvious thing here and just do two checks.

Also, I should fix the other borders too, but please do the calculations in local space rather than device space. Thanks!